test-avocado/lib/testvm.py

```diff
@@ -0,0 +1 @@
+../../test/testvm.py
```

Comment:
Yes,
* this is first option
* second option is to add also Class MachineClass to try: block beucase it is still tracebacking, althounght I'm for example importing only Browser class to tests.
* or probaly the most clean option is to split testlib.py to separated modules It will solve more issue 
  * let Browser in one file, MachineClass into another one and others helper functions to some another file.